OpenRA.Game/Graphics/CursorSequence.cs

```diff
@@ -31,19 +31,18 @@ public CursorSequence(FrameCache cache, string name, string cursorSrc, string pa
 			Palette = palette;
 			Name = name;
 
+			Frames = cache[cursorSrc].Skip(Start).ToArray();
+
 			if ((d.ContainsKey("Length") && d["Length"].Value == "*") || (d.ContainsKey("End") && d["End"].Value == "*"))
-				Length = Frames.Length - Start;
+				Length = Frames.Length;
 			else if (d.ContainsKey("Length"))
 				Length = Exts.ParseIntegerInvariant(d["Length"].Value);
 			else if (d.ContainsKey("End"))
-				Length = Exts.ParseIntegerInvariant(d["End"].Value) - Start;
+				Length = Exts.ParseIntegerInvariant(d["End"].Value);
```

Comment:
Sorry, but on second thought the answer to my question "It looks like the End case will also have the same problem?" is "No, maths doesn't work like that". We do need the `Start` here because End is specified relative to full frame count, not the skipped array.

Reply:
Fixed it again.